test/web-platform-tests/to-upstream/html/semantics/embedded-content/the-img-element/img.complete-2.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
Can you name this `-dont-upstream.html` since it's a copy of an existing file and we need to remember not to upstream it?

Reply:
Hmm I see it's not a copy of an existing file... I'll give this some thought.